extensions/wikibase/tests/data/manifest/wikidata-manifest-warning-data-test.json

```diff
@@ -0,0 +1,92 @@
+{
+    "version": "1.0",
+    "mediawiki": {
+        "name": "Wikidata",
+        "root": "https://www.wikidata.org/wiki/",
```

Comment:
Do we need a new copy of the Wikidata manifest, can't we reuse the existing ones we have in this folder?

Reply:
Yes we can do that; removing the added manifest file. Thanks.